### PR TITLE
Home.md

### PR DESCRIPTION
**Pull request by SonyaMoisset**
French Translation of Home.md

### CODE REVIEW

**Comment by Rafase282** [comment]
Thanks for the article. I don't speak French but given that Google insist on translating the page when I load it, it seems to be fine. Anyone from @FreeCodeCamp/wiki-mods speaks French?

File: fr.lang/Home.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+### Guides sur comment contribuer
+- [Comment contribuer à partir de votre navigateur](Guide-Online-Contribution): Ceci est actuellement la meilleure solution pour contribuer. Vous pouvez lire cet [article](https://medium.freecodecamp.com/how-to-land-your-first-open-source-contribution-from-your-browser-in-15-minutes-756d9bbf81ad) pour plus de détails.
+- [Guide sur les Pull Request](PULL_REQUEST_TEMPLATE)
+- [Comment faire pour contribuer au Wiki:](How-To-Contribute-To-The-Wiki) Ceci est un guide qui va vous montrer comment obtenir une copie locale de sorte que vous puissiez apporter des modifications à partir de votre ordinateur, il fonctionne également pour contribuer au dépôt principal.
+- [En savoir plus sur la façon de faire une pull request à partir d'une fork](Pull-Request-Contribute)
+- [Comment Squash vos Commits:](git-rebase # squashing-multiple-commits-en-un) Nous exigeons un seul commit par pull request, sauf indication contraire émise par un modérateur.
+
+Si vous avez des questions à ce sujet, veuillez envoyer un message à [Rafase282 dans Gitter](https://gitter.im/Rafase282) ou rejoignez le [salon Gitter Wiki](Https://gitter.im/FreeCodeCamp/Wiki) pour en discuter.
```
Comment (hallaathrad) [inline fr.lang/Home.md]: peut-être préciser qu'il ne parle pas français?

**Comment by hallaathrad** [comment]
With that little change I've suggested, ça marche pour moi :+1:

**Comment by Rafase282** [comment]
Thanks for that, didn't come to mind that they would be messaging me in French.

**Pull request merged**